.github/workflows/release.yml

```diff
@@ -123,3 +130,51 @@ jobs:
           chmod 0600 ~/.gem/credentials
           gem signin
           for gem in pkg/*.gem; do gem push $gem -V; done
+
+  verify:
+    name: Verify published gem on ${{ matrix.os }}
+    needs: publish
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+    steps:
```

Comment:
It makes sense to add some sleep step here for "rubygem cooldown", to make sure last pushed gem is available through rubygems

From my own experience sometimes `gem install` right after `gem push` cannot get last pushed gem

Reply:
Thank you, that shall help